recipe/meta.yaml

```diff
@@ -1,9 +1,8 @@
 {% set name = "mamba" %}
-{% set libmamba_version = "1.4.4" %}
-{% set libmambapy_version = "1.4.4" %}
-{% set mamba_version = "1.4.4" %}
-{% set release = "2023.05.16" %}
-
+{% set libmamba_version = "1.4.2" %}
+{% set libmambapy_version = "1.4.2" %}
+{% set mamba_version = "1.4.2" %}
+{% set release = "2023.04.06" %}
```

Comment:
Can we remove this release? It is not used anywhere in the recipe.

Reply:
Sure, Will update the recipe accordingly.